src/decide.ts

```diff
@@ -63,9 +64,11 @@ function compareFileTrees(candidate: Metafile, searchee: Searchee): boolean {
 }
 
 function sizeDoesMatch(resultSize, searchee) {
+	const fileConfig = getRuntimeConfig();
```

Comment:
fileConfig and runtimeConfig are two separate things. Please rename this to runtimeConfig